docs/development/releasing.rst

```diff
@@ -6,17 +6,26 @@ This page describes the release process for the core astropy package. For the av
 coordinated or affiliated package, you can instead check
 :ref:`these <simple-release-docs>` instructions which will be a lot simpler.
 
-The lifetime of a major release cycle of the core package is as follows:
+The core package has adopted the following release plan:
+
+* Major releases every two years. These are releases that can introduce breaking API changes, and should be numbered as x.0.0 (e.g., 6.0.0).
```

Comment:
```suggestion
* Major releases every year (or longer). These are releases that can introduce breaking API changes, and should be numbered as x.0.0 (e.g., 6.0.0).
```

See https://github.com/astropy/astropy-APEs/blob/ccdb92ea37220c67a5524791e7fb7626f54f49b3/APE2.rst?plain=1#L24  which was updated as part of APE 21